test/navigation/NavigationSpec.scala

```diff
@@ -64,7 +64,8 @@ class NavigationSpec extends SpecBase {
         (FileListPage, controllers.routes.OtherReasonController.onPageLoadForUploadEvidenceQuestion(CheckMode).url),
         (UploadEvidenceQuestionPage, controllers.routes.CheckYourAnswersController.onPageLoad().url),
         (AppealSinglePenaltyPage, controllers.routes.PenaltySelectionController.onPageLoadForPenaltySelection(CheckMode).url),
-        (AppealCoverBothPenaltiesPage, controllers.routes.PenaltySelectionController.onPageLoadForPenaltySelection(CheckMode).url)
+        (AppealCoverBothPenaltiesPage, controllers.routes.PenaltySelectionController.onPageLoadForPenaltySelection(CheckMode).url),
+        (AppealByLetterKickOutPage, controllers.routes.CancelVATRegistrationController.onPageLoadForCancelVATRegistration().url)
```

Comment:
This page won't be available in checkmode as it's a kick out page (e.g. users can't 'check' their answers on this page). Could you remove this? :sorry: